src/store/trade.store.ts

```diff
@@ -40,10 +39,11 @@ export class TradeStore {
   }
 
   async update(selector: string, newTrades: TradeItem[]) {
+    newTrades.forEach((value) => value['selector'] = selector)
```

Comment:
Not really a nice way of doing this. You're basically mutating the original object.

```
const tradesCollection = newTrades.map((trade) => ({ ...trade, selector }))
```